Examples/Java/Sources/Everything.java

```diff
@@ -25,30 +26,6 @@
 import java.util.Objects;
 import java.util.Set;
 
-interface EverythingMapPolymorphicValuesMatcher<R> {
-    R match(@Nullable User value0);
-    R match(@Nullable Board value1);
-    R match(@Nullable Image value2);
-    R match(@Nullable Pin value3);
-    R match(@Nullable Everything value4);
-    R match(@Nullable List<Object> value5);
-    R match(@Nullable Map<String, Object> value6);
-}
-
-interface EverythingPolymorphicPropMatcher<R> {
-    R match(@Nullable User value0);
-    R match(@Nullable Board value1);
-    R match(@Nullable Image value2);
-    R match(@Nullable Pin value3);
-    R match(@Nullable Everything value4);
-    R match(@Nullable String value5);
-    R match(@Nullable Boolean value6);
-    R match(@Nullable Integer value7);
-    R match(@Nullable Double value8);
-    R match(@Nullable Date value9);
-    R match(@Nullable String value10);
```

Comment:
This second declaration of `match(String)` is a bug that I did not solve in this PR. It happens because the schema has two string types declared in the oneOf:
```
    "polymorphic_prop": {
      "oneOf" : [
        { "$ref" : "user.json" },
        { "$ref" : "board.json" },
        { "$ref" : "image.json" },
        { "$ref" : "pin.json" },
        { "$ref" : "everything.json" },
        { "type" : "string" },
        { "type" : "boolean" },
        { "type" : "integer" },
        { "type" : "number" },
        { "type" : "string", "format": "date-time" },
        { "type" : "string", "format": "uri" }
      ]
    },
```